website/cue/reference/releases/0.33.0.cue

```diff
@@ -0,0 +1,691 @@
+			pr_numbers: [18609]
+			breaking: true
+		},
+		{
+			type: "enhancement"
+			scopes: ["datadog_agent source", "datadog_metrics sink"]
+			description: """
+				The `datadog_metrics` sink now transmits the origin of the metric to Datadog for display
+				in metrics explorer. Metrics from the `datadog_agent` source have their origin passed
+				through but otherwise an appropriate origin header is set for other metrics sources.
+				"""
+			pr_numbers: [18405]
+		},
+		{
+			type: "enhancement"
+			scopes: ["codec"]
+			description: """
+				Vector's protobuf definition for events was updated to have a new field to store
+				metadata. Most users will not care about this, but if you are consuming Vector's
+				protobuf encoded events directly please see [upgrade
+				guide](/highlights/2023-09-26-0-33-0-upgrade-guide#vector-proto-metadata).
+				"""
+			pr_numbers: [18405]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kafka sink"]
+			description: """
+				The `kafka` sink now builds requests concurrently which is expected to improve
+				performance.
+				"""
+			pr_numbers: [18634]
+		},
+		{
+			type: "fix"
+			scopes: ["sinks"]
+			description: """
+				The following sinks now avoid ballooning in memory consumption in the precense of
+				back-pressure by limiting request building concurrency:
+
+				- `amqp`
+				- `appsignal`
+				- `azure_monitor_logs`
+				- `clickhouse`
+				- `gcp_stackdriver`
+				- `kafka`
+				- `honeycomb`
+				- `http`
+				- `nats`
+				- `pulsar`
+				"""
+			pr_numbers: [18634, 18637]
+		},
+		{
+			type: "enhancement"
+			scopes: ["journald source"]
+			description: """
+				The `journald` source now has an `extra_args` configuration option to allow
+				specifying additional arguments to pass through to `journalctl` when fetching
+				events.
+				"""
+			pr_numbers: [18568]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl"]
+			description: """
+				VRL was updated to v0.7.0 which brings the following:
+
+				Bug Fixes:
+
+				- `parse_nginx_log` doesn't fail if the values of key-value pairs in error logs are missing
+				- `encode_gzip` and `encode_zlib` now correctly check that the compression level is valid, preventing a panic
+				- type definitions for arrays and objects with undefinide values was improved
+				- `parse_aws_vpc_flow_log now` handles account-id value as a string, avoiding loss of leading zeros and handling the case where value is unknown
+
+				Features:
+
+				- `parse_key_value` can now parse values enclosed in single quote characters
+				- added `pretty` parameter for `encode_json` function to produce pretty-printed JSON string
+				- added `community_id` function for generation of [Community IDs](https://github.com/corelight/community-id-spec)
+				- `parse_aws_vpc_flow_log` can now handle VPC logs using version 5 fields
+				- the deprecated `to_timestamp` function was removed (use `parse_timestamp` and `from_unix_timestamp` instead)
+				"""
+			pr_numbers: [999999]
```

Comment:
TODO: replace with actual PR number updating. I just put this placeholder here to fill out the other details in advance of the VRL upgrade.